test/unit/outFile.spec.ts

```diff
@@ -0,0 +1,113 @@
+import * as util from "../util";
+import * as TSTLErrors from "../../src/TSTLErrors";
+
+const exportValueSource = "export const value = true;";
+const reexportValueSource = 'export { value } from "./export";';
+const exportResultSource = "export const result = value;";
```

Comment:
`exportValueSource` and `exportResultSource` have similar names, but assign different values. I'm not really sure there's a need for these to be variables, it might be easier to read with it inlined.